metaseq/models/transformer_decoder.py

```diff
@@ -473,7 +474,7 @@ def max_positions(self):
         return min(self.max_target_positions, self.embed_positions.max_positions)
 
     def buffered_future_mask(self, tensor, input_tokens=None):
-        batch_size, cur_seq_len = tensor.size(0), tensor.size(1)
+        cur_seq_len, batch_size = tensor.size(0), tensor.size(1)
```

Comment:
Oh noes.. lost me here - how did this flip?

Reply:
ohh cause, I am not converting to T x B x C before this call